packages/neuron-ui/src/widgets/Pagination/index.tsx

```diff
@@ -42,32 +44,38 @@ const Pagination = ({ count, pageNo, onChange, pageSize }: PaginationProps) => {
   const end = Math.min(count, pageNo * pageSize)
   const range = t(`${I18N_PATH}.range`, { start, end, count })
 
+  const disableToPrev = pageNo === 1 || pageCount === 0
+  const disableToEnd = pageNo === pageCount || pageCount === 0
```

Comment:
```suggestion
  const disableToEnd = pageNo >= pageCount
```

Reply:
`disableToPrev` and `disableToEnd` are not paired, `Prev/Next` or `Head/End` would be better